exchange/utils.go

```diff
@@ -167,19 +167,19 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 		// apply bidder-specific schains
 		sChainWriter.Write(reqWrapperCopy, bidder)
 
+		// eid scrubbing
+		if err := removeUnpermissionedEids(reqWrapperCopy, bidder); err != nil {
+			errs = append(errs, fmt.Errorf("unable to enforce request.ext.prebid.data.eidpermissions because %v", err))
+			continue
+		}
+
```

Comment:
`removeUnpermissionedEids` function should be executed before `buildRequestExtForBidder` because it needs access to `req.ext.prebid.data.eidpermissions`. 

`buildRequestExtForBidder` function rebuilds `req.ext` and doesn't add `req.ext.prebid.data.eidpermissions`.

Reply:
I think you're right. It looks like we just need to read from `req.ext.prebid.data` and copy any applicable EIDs to `req.user` and then we can discard `req.ext.prebid.data`. If that's the case then we should move `removeUnpermissionedEids` just before `buildRequestExtForBidder` since `buildRequestExtForBidder` will scrub `req.ext.prebid.data`.